e2e_tests/docker_image/Dockerfile

```diff
@@ -22,23 +22,21 @@ RUN cd tpm2-tss \
 	&& make -j$(nproc) \
 	&& make install \
 	&& ldconfig
-
 RUN rm -rf tpm2-tss
 
-ARG ibmtpm_name=ibmtpm1637
```

Comment:
If we could update the upstreamed Dockerfile with the changes in #381 that'd be great 👌